src/main/java/fi/crosskey/card/vehicle/simulator/models/Helicopter.java

```diff
@@ -0,0 +1,78 @@
+package fi.crosskey.card.vehicle.simulator.models;
+
+import javax.persistence.*;
```

Comment:
In general, we want to stay away from importing with `*`, but it's no biggie here.